capella2polarion/converters/model_converter.py

```diff
@@ -112,10 +112,6 @@ def generate_work_items(
             generate_attachments,
         )
         work_items = serializer.serialize_all()
-        for work_item in work_items:
-            assert work_item.title is not None
-            assert work_item.type is not None
-
```

Comment:
why? This should never fail and if it does, we should not even start pushing changes to Polarion

Reply:
Why do we need to iterate over all serialized `work_items` again? Why not do the assertions after the serialization immediately? See here:
https://github.com/DSD-DBS/capella-polarion/pull/134/files#diff-4d92078b5ac786b0daad60e9a6c0a23e9c57c5cd3eb774696e56d19f91677ef5R111-R114